### ISSUE TITLE
[A11Y]: <amp-video>, <amp-audio> should propagate `aria-*` to the inner `<video/audio>`

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#5493]
Currently
`<amp-video aria-label="Some Video"  ..` does NOT read "Some Video" when using screen readers the same way `<video aria-label="Some Video" ...` does.

Same is true for other `aria-*` attributes.

We need to propagate all `aria-*` and `role` attributed from the parent `<amp-video>` to `<video>` since the `<video>` is the real component here.

**To be on the safe side**, we can start by only propagating `aria-label`, `aria-labelledby` and `aria-describedby`

**Comment by aghassemi** [comment]
@mkhatib pretty much same thing as #5489 so you get two issues :)

**Comment by aghassemi** [comment]
https://github.com/ampproject/amphtml/pull/5590 closes this.
